configuration/scripts/ice_in

```diff
@@ -613,6 +614,8 @@
     f_trsig        = 'm'
     f_icepresent   = 'm'
     f_iage         = 'm'
+    f_icedir       = 'x'
+    f_icespd       = 'x'
```

Comment:
Why are these here, between the two age flags?  It would make more sense for them to reside next to uvel, vvel or divu.